site/cypress/e2e/polls.spec.js

```diff
@@ -149,6 +149,8 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('#poll-date').clear({force: true});
         cy.get('#poll-date').type('1970-01-01', {force: true});
         cy.get('h1').click(); // get rid of the date picker
+        cy.get('#student-histogram-release-setting').select('never');
+        cy.get('#student-answer-release-setting').select('never');
```

Comment:
If you're creating a new poll and these are default values, there's no need to set them.  Furthermore, this would be a good chance to make sure the default values are set correctly.

I can't comment on unchanged lines so I'll leave my comment here:
It might be good to add some checks for the release histogram and answer release setting a little bit after line 186.  Adding checks around line 212 might be good for the same reason.